.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: waiting-for-triage
+assignees: ''
+
+---
+
+**Describe the bug**
+Please provide details of the problem, including the version of Spring Cloud that you
```

Comment:
```suggestion
Please provide details of the problem, including the version of Spring Cloud GCP that you
```

Reply:
The version of Spring Cloud would correlate with the version of GCP.  Some people might not know how to figure out the exact project version